### DISCUSSION TITLE
Text selection is (still) not working for dark themes

### DISCUSSION THREAD

**Discussion by nick-delirium** [community/community#55544] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Select Feature Area

- [X] Issues
- [X] Projects

### Body

macOS, Chrome (latest),

text selection pretty much everywhere on the website is not visible while using dark themes. I even traced down the issue to this css rule. Disabling it (thats what I'm doing currently via injected script) makes things work normally.

![image](https://github.com/community/community/assets/23126999/fc0534d1-77df-4c0c-aff4-1af1c6cdd8b4)

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @nick-delirium ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!
